scripts/packages/VSCodeServer/src/misc.jl

```diff
@@ -11,8 +11,21 @@ function find_frame_index(bt::Vector{<:Union{Base.InterpreterIP,Ptr{Cvoid}}}, fi
     end
     return
 end
-
-
+function find_first_topelevel_scope(bt::Vector{<:Union{Base.InterpreterIP,Ptr{Cvoid}}})
+    for (i, ip) in enumerate(bt)
+        st = Base.StackTraces.lookup(ip)
+        ind = findfirst(st) do frame
+            if frame.linfo isa Base.CodeInfo
+                if frame.linfo.linetable[1].method == Symbol("top-level scope")
+                    return true
+                end
+            end
```

Comment:
nit, just improve robustness:
```suggestion
            linfo = frame.linfo
            if linfo isa Core.CodeInfo
                linetable = linfo.linetable
                if isa(linetable, Core.LineInfoNode) && length(linetable) ≥ 1 && first(linetable).method === Symbol("top-level scope")
                    return true
                end
            end
```